Automattic/wp-calypso
# Tailored flows: NL - reorder plans features

#### Proposed Changes

* This does the tasks listed here: https://github.com/Automattic/wp-calypso/issues/68216

#### Testing Instructions

1. Go to /setup/?flow=newsletter.
2. Reach the plans grid.
3. The features shouldn't have `Import subscribers`.
4. The features shouldn't have `Google analytics`.
5. The features shouldn't have `Unlimited themes`.
6. Highlighted features should be first.

Fixes: https://github.com/Automattic/wp-calypso/issues/68216
Fixes: https://github.com/Automattic/wp-calypso/issues/68215
